From de72886e9df6647e3e4a0b2980e8335f631d854e Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Mon, 19 Jan 2026 20:40:09 +0200
Subject: [PATCH 2/2] mrvl-prestera update 15jan Nokia

intDriver: Add timeout WA for lost IRQs (sonic-net#56)
sonic-ext.target wrapper for sonic.target (sonic-net#60)
amd64: ethDriver.c force DMA_BIT_MASK 32bits (sonic-net#59)

Signed-off-by: Yan Markman <ymarkman@marvell.com>
---
 platform/marvell-prestera/mrvl-prestera | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/marvell-prestera/mrvl-prestera b/platform/marvell-prestera/mrvl-prestera
index 0e6334897..52cdc81db 160000
--- a/platform/marvell-prestera/mrvl-prestera
+++ b/platform/marvell-prestera/mrvl-prestera
@@ -1 +1 @@
-Subproject commit 0e63348973df0c1793684704cb6e9d07bf5d473e
+Subproject commit 52cdc81db3585515f9c16be2c295e9c2d5f0b3ea
-- 
2.34.1

